### DISCUSSION TITLE
An error occurs in the fit method of glm single. Please let me know how to resolve this.

### DISCUSSION THREAD

**Discussion by 265091z** [cvnlab/GLMsingle#113] [Q&A]
hello

I found your paper and thought GLMSingle was a great idea.
I am new to "GLMSingle" and am having trouble using it and would like your opinion.

Study design
Each participant will perform 5 lanscans with fMRI. Participants will be presented with 12 different stimuli for 2 seconds.

Input to GLMSingle
<Design>
Length of Design list: 5
Design[0].shape：(697, 2) ＜ー (TR, conditions[groupA, groupB])
Design[0].type：float64のnumpy.ndarray
<fMRI Data>
Length of list of Data: 5
Data[0].shape：(97, 115, 97, 697)
Data[0].type：float32のnumpy.ndarray

<code>
results_glmsingle = glmsingle_obj.fit(
design matrix for 5runs of one subject, the
5runs of fmri data for one subject, #stimulus duration
2, #stimulus duration
0.743, #TR
outputdir=OUTPUTDIR
)

<error>
glmsingle.py (line 1100) glmbadness[relix, : ] = calcbadness()
resultsdm[runis][:, whcol] = zerodiv(rundemean, sd, val=0, wantcaution=0)
ValueError: shape mismatch: value array of shape(0,) could not be broadcast to indexing result of shape (89, 0)

Q: How can I resolve the error?

**Comment by kendrickkay** [comment]
I believe this is a known bug. @iancharest ?

**Reply by 265091z** [reply]
Sorry for the lack of study. I could not find a way to deal with this error. Could you please provide a link to the page? Thank you in advance for your help.

**Reply by kendrickkay** [reply]
We are aware of the issue and are actively working on it...
